docs/user/FitTutorial.rst

```diff
@@ -0,0 +1,279 @@
+same and lower and upper times are the same.
+
+.. code:: python
+
+    data_cnt = 60
+    avgints = pd.DataFrame(dict(
+        integrand="susceptible",
+        location=parent_location,
+        age_lower=np.linspace(0, 120, data_cnt),
+        age_upper=np.linspace(0, 120, data_cnt),
+        time_lower=2000,
+        time_upper=2000,
+    ))
+    avgints = pd.concat([avgints, avgints.assign(integrand="mtother")])
+
+Those are the inputs for running Dismod-AT predict. The next step is to
+start a Dismod-AT session and run predict. A Dismod-AT session
+communicates with Dismod-AT running underneath.
+
+.. code:: python
+
+    session = Session(locations, parent_location, Path("fit0.db"))
+    session.set_option(ode_step_size=1)
+
+Note the ``ode_step_size=1``. This is the step size for predict. We can
+play with this to understand how step size affects fit accuracy, because
+a Dismod-AT fit is built upon running predict over and over again during
+optimization.
+
+Prediction returns two sets of data, the data points requested and those
+that *could not be predicted* because they were associated with
+covariates that are outliered. No such covariates are in this model, so
+that list will be empty.
+
+.. code:: python
+
+    predicted, not_predicted = session.predict(model_variables, avgints, parent_location)
+    assert not_predicted.empty and not predicted.empty
+    print(f"Predicted outcome with columns {', '.join(str(pc) for pc in predicted.columns)}")
+
+
+.. parsed-literal::
+
+    Predicted outcome with columns sample_index, avg_integrand, location, integrand, age_lower, age_upper, time_lower, time_upper
+
+
+The predictions are a Pandas DataFrame with the columns shown. They will
+match the avgints, so there are two sets of outcomes in this case,
+mtother and susceptibles. This is now our data.
+
+Let's fit that data. This entails making a Model and setting its priors.
+
+.. code:: python
+
+    nonzero_rates = ["omega"]
+    model = Model(nonzero_rates, parent_location)
+    model.rate["omega"] = SmoothGrid(np.linspace(0, 120, age_points), [2000])
+    for a, t, dage, dtime in model.rate["omega"].age_time_diff():
+            target = omega(a, t)
+            model.rate["omega"].value[a, t] = Uniform(mean=target, upper=target + 5, lower=0)
+            model.rate["omega"].dage[a, t] = Gaussian(mean=0, standard_deviation=50, lower=-5, upper=5)
+            model.rate["omega"].dtime[a, t] = Gaussian(mean=0, standard_deviation=0.1, lower=-5, upper=5)
+
+That's the whole model. We could set the standard deviation on the
+age-difference priors from the forward age difference if we wanted, but
+it's enough to let them have a very large standard deviation.
+
+While that was 8 lines, the output of predict needs some help to look
+like input data.
+
+.. code:: python
+
+    data = predicted.drop(columns=["sample_index"]) \
+        .rename(columns={"avg_integrand": "mean"})
+    data = data.assign(density="gaussian", std=0.3, eta=1e-4, nu=nan)
```

Comment:
I think it's  worth explaining what this line does. Maybe it will be self evident to a modeler...